underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
To support the `count = undefined` case and fix the `count = 1` case, we should default to `0` here.